spring-cloud-starter-stream-source-twitterstream/src/main/java/org/springframework/cloud/stream/app/twitterstream/source/AbstractTwitterInboundChannelAdapter.java

```diff
@@ -227,16 +228,16 @@ private void waitLinearBackoff() {
 		}
 	}
 
-	private void waitRateLimitBackoff() {
+	private void waitRateLimitBackoff(HttpStatusCodeException sce) {
 		int millis = rateLimitBackOff.get();
```

Comment:
From the issue...

> and expose properties for all those back-offs.

Did you decide that was not appropriate for some reason?